src/main/java/uk/co/ramp/utilities/UtilitiesBean.java

```diff
@@ -10,19 +10,6 @@
 import java.util.stream.Stream;
```

Comment:
Some unused imports.

Reply:
Actually, it might be worth binning this class and having this as a static method on Case.

Reply:
Binned this class and moved the getMostSevere method into ContactEventProcessor as a private method, as it is only place it is used within the code base